spec.html

```diff
@@ -17760,6 +17779,121 @@ <h1>EnumerateObjectProperties ( _O_ )</h1>
             }
           </code></pre>
         </emu-note>
+        <emu-note>
+          The list of exotic objects for which hosts are not required to match CreateForInIterator was chosen because hosts historically differed in behavior for those cases, and agreed in all others.
+        </emu-note>
+      </emu-clause>
+
+      <emu-clause id="sec-for-in-iterator-objects">
+        <h1>For In Iterator Objects</h1>
```

Comment:
would it be worth saying "For-In" for clarity that it's a single token? i feel like i've commonly seen this representation when describing the construct, to disambiguate from prose.